phase-4/update-node.md

```diff
@@ -0,0 +1,82 @@
+# Update your validator node
+:::tip Rewards  
+Upon completing this challenge, you will be rewarded **50 Desmos Tokens**. 
+  
+Additionally, you will also earn more tokens the longer you keep the node running. To know more about this please reference the [Validators program](validators-program/overview.md).   
+:::
+
+When a new version of Desmos is released, all validators need to update their node so that it can keep running properly. Following you will find the guide on how you can do this making sure everything is ready for when the new chain is started. 
+
+## Quick update
+The following guide is based on the genesis file we have already published on our GitHub repository. If you feel comfortable with what we have done, you can follow the below steps. On the other hand, if you want to check that what we have done is correct and not malicious, you can perform a [manual update](#manual-update) instead. 
+
+In order to perform the quick update, execute the following commands. 
+
+1. Stop the running service:   
+   ```bash
+   sudo systemctl stop desmosd
+   ```
+   
+2. Update the Desmos binaries:  
+   ```bash
+   cd ~/desmos
+   git fetch -a 
+   git checkout tags/v0.4.0
+   make install
+   ```
+   
+3. Delete the current `genesis.json` file and download the new one:  
+   ```bash
+   rm ~/.desmosd/config/genesis.json
+   curl https://raw.githubusercontent.com/desmos-labs/morpheus/master/genesis.json > ~/.desmosd/config/genesis.json
+   ```
+   
```

Comment:
May suggest them to verify with the `sha256` hash of the genesis file which will be provided later.